### PR TITLE
various: Standardize add-on vs addon in help

### PR DESCRIPTION
**Pull request by kingthorin**
## Overview
- CHANGELOG > Add maint note.
- various html > Updated spelling.

## Related Issues
na

## Checklist
- [x] Update help
- [x] Update changelog
- [na] Run `./gradlew spotlessApply` for code formatting
- [na] Write tests
- [na] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

### CODE REVIEW

File: addOns/accessControl/src/main/javahelp/org/zaproxy/zap/extension/accessControl/resources/help/contents/concepts.html [hunk 1]
```diff
@@ -97,7 +97,7 @@ <H3 id="id-10102">Access Control Issue - Improper Authorization</H3>
 
 <H2>API</H2>
 
-The Addon exposes the following API endpoints:
+The Add-on exposes the following API endpoints:
```
Comment (thc202) [inline addOns/accessControl/src/main/javahelp/org/zaproxy/zap/extension/accessControl/resources/help/contents/concepts.html:100]: Does not need capitalization IMO.

**Comment by thc202** [comment]
`grpc` and `zest` should be changed as well.

File: addOns/pscanrules/src/main/javahelp/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html
Comment (thc202) [inline addOns/pscanrules/src/main/javahelp/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html:1]: Update the occurrence under Information Disclosure: Suspicious Comments.

Comment (kingthorin) [reply addOns/pscanrules/src/main/javahelp/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html:1]: Done

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Pull request merged**